### DISCUSSION TITLE
Everything setup, how do I start customizing it?

### DISCUSSION THREAD

**Discussion by GabrielRybak** [philomena-dev/philomena#217] [Q&A]
Hello, I'm currently learning some docker and server setups so this is an interesting project. I wanted to know if there is any more documentation or is this all just try and error.
I already have everything setup, but I don't know how to further advance in this, for example, setting a path directory for already loaded media content, or how to delete customize the board itself.

**Comment by liamwhite** [comment]
The documentation is minimal because I got busy and was not able to continue working on it for a long time. I am in the process of fixing things up and adding the missing documentation but it will likely take a while.

I am not sure what you mean by "already loaded media content" or "how to delete customize the board" -- can you elaborate?
